docs/products/mysql/howto/create-remote-replica.rst

```diff
@@ -0,0 +1,26 @@
+Create MySQL remote replica
+===========================
+
+The goal of the remote replica is to provide a read-only instance of your managed MySQL service in another geographically diverse region. 
+
+This works as an extra measure to protect your data from the unlikely event that a whole region would go down. It can also improve performance if a read replica is placed closer to your end-users that read from the database.
+
+Here are the steps to provision remote replica
```

Comment:
Here are the steps to provision remote replica => Take the following steps to provision a remote replica: